.travis.yml

```diff
@@ -26,97 +27,15 @@ addons:
 
 rvm:
   - 2.4.1
-  - 2.3.4
-  - 2.2.7
-  - 2.1.10
-  - 2.0.0
```

Comment:
This is the version that ships with Mac, would it make sense to keep this right now?